Mlem/App/Views/Shared/Search/SearchView+FiltersView.swift

```diff
@@ -0,0 +1,217 @@
+                        personFiltersView
+                    case .instances:
+                        instanceFiltersView
+                    }
+                }
+                .padding(.vertical, 12)
+                .padding(.horizontal, Constants.main.standardSpacing)
+            }
+            .scrollIndicators(.hidden)
+            if selectedTab == .communities, communityFilters.instance.isOther {
+                Label(
+                    "Subscription statuses can't be displayed when using these filters.",
+                    systemImage: Icons.warning
+                )
+                .font(.footnote)
+                .foregroundStyle(palette.accent)
+                .padding(.bottom, 12)
+                .padding(.horizontal, Constants.main.standardSpacing)
+            }
+        }
+        .background(palette.accent.opacity(0.1))
+        .animation(.easeOut(duration: 0.1), value: filterAnimationHashValue)
+    }
+    
+    @ViewBuilder
+    private var communityFiltersView: some View {
+        FeedSortPicker(
+            sort: $communityFilters.sort,
+            filters: [.availableOnInstance, .communitySearchable]
+        )
+        .buttonStyle(FilterButtonStyle(isOn: communityFilters.sort != .topAll))
+        InstancePicker(filter: $communityFilters.instance, isForPersonSearch: false)
+            .buttonStyle(FilterButtonStyle(isOn: communityFilters.instance != .any))
+    }
+    
+    @ViewBuilder
+    private var personFiltersView: some View {
+        FeedSortPicker(
+            sort: $personFilters.sort,
+            filters: [.availableOnInstance, .personSearchable]
+        )
+        .buttonStyle(FilterButtonStyle(isOn: personFilters.sort != .topAll))
+        InstancePicker(filter: $personFilters.instance, isForPersonSearch: true)
+            .buttonStyle(FilterButtonStyle(isOn: personFilters.instance != .any))
+    }
+    
+    @ViewBuilder
+    private var instanceFiltersView: some View {
+        Menu(
+            String(localized: instanceFilters.sort.label),
+            systemImage: instanceFilters.sort.systemImage
+        ) {
+            Picker("Sort", selection: $instanceFilters.sort) {
+                ForEach(InstanceSort.allCases, id: \.self) { sort in
+                    Label(String(localized: sort.label), systemImage: sort.systemImage)
+                }
+            }
+            .pickerStyle(.inline)
+        }
+        .buttonStyle(FilterButtonStyle(isOn: instanceFilters.sort != .score))
+    }
+    
+    private struct InstancePicker: View {
+        @Environment(NavigationLayer.self) var navigation
+        @Binding var filter: InstanceFilter
+        let isForPersonSearch: Bool
+        
+        var allowActiveAccountLocalInstanceSearch: Bool {
+            !isForPersonSearch || (AppState.main.firstApi.fetchedVersion ?? .infinity) >= .v19_4
+        }
+        
+        var body: some View {
+            Menu("Instance: \(filter.label)", systemImage: Icons.instance) {
```

Comment:
I don't think we need the explicit "Instance:" text--the icon should make the behavior reasonably evident, and it'll be very obvious once the user taps it once. We also don't have any additional text on the sort selector, so removing "Instance:" would be consistent.

Reply:
Sure, that's a good idea 👍